charts/c100-application/values.yaml

```diff
@@ -45,3 +45,7 @@ base:
           alias: MAINTENANCE_ALLOWED_IPS
 redis:
   enabled: false
+useInterpodAntiAffinity: true
+keyVaults:
+  c100:
+    anything: here
```

Comment:
these should all be nested under base

Reply:
"hmcts.interpodantiaffinity.v2"  requires useInterpodAntiAffinity at the root level.
"hmcts.secretCSIVolumes.v2" requires keyvaults: etc at the root level